zadaci/sa-kolokvijuma/2020/RA/T5/G8/index.rst

```diff
@@ -0,0 +1,18 @@
+.. index::
+   Algoritmi ; Pretraga
+   Datoteke ; Tekstualne
+   Dinamičke strukture ; Lista
+
+
+Zadatak "Balet"
```

Comment:
Please give more descriptive name, for example "Rezervacija karata za baletsku predstavu"

Reply:
Legth of "====" below title must match the length of a title, bad examples:

```
Naslov
==============
```

```
Naslov
===
```

Good example:

```
Naslov
======
```